doc/releases/release-notes-2.1.rst

```diff
@@ -96,6 +96,11 @@ Build and Infrastructure
   Use new functions that utilize eDTS info such as dt_node_reg_addr.
   See :zephyr_file:`scripts/kconfig/kconfigfunctions.py` for details.
 
+* Deprecated direct use of the `DT_` Kconfig symbols from the generated
```

Comment:
Needs two backticks around `DT_` (and similarly in other places).

Reply:
yep, single backticks tell Sphinx to "hunt" for a matching label reference (e.g., for a function or inter-document labels defined with a ``.. _label_here:`` notation).  Use double backticks to get a formatted filename reference (equivalent to `` :file:`filename` ``)

Reply:
what should I do since `DT_` isn't meant to be a file name reference, just something that we'd want inline code style formatting for.?

Reply:
Just two backticks will do it in that case.